app-backend/datamodel/src/main/scala/com/azavea/rf/datamodel/Project.scala

```diff
@@ -19,6 +21,9 @@ case class Project(
   description: String,
   visibility: Visibility,
   tileVisibility: Visibility,
+  isAOIProject: Boolean,
+  aoiCadence: Long, /* Milliseconds */
```

Comment:
Kind of why I started to wonder again if it makes sense to move the cadence and last checked to AOI instead of the project, as that may be the most correct even if that complicates the checking logic.